CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Current
 
 ### Added:
 
+- [Evaluate format type from both URI and Accept header](https://github.com/yahoo/fili/pull/495)
+    * Add a new functional interface `ResponseFormatResolver`.
+    * Implement a concrete implementation of `ResponseFormatResolver` in `AbstractBindingFactory`.
+
```

Comment:
These notes describe where the changes are, but we should have something in here about the nature of the change (e.g. mapping Accept headers and coalescing with URI defined headers)